SoftLayer/CLI/registration/show.py

```diff
@@ -0,0 +1,29 @@
+"""Display the RWhois information for your account."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """Display the RWhois information for your account."""
+
+    mgr = SoftLayer.NetworkManager(env.client)
+    result = mgr.get_registrations()
+    table = formatting.KeyValueTable(['id', 'createDate', 'networkIdentifier',
```

Comment:
This should just be a normal Table, not `KeyValueTable`

When doing JSON output this will only print the first 2 values:

```
$ ./slcli --format=json registration   show
{
    "83524": "2014-06-23 12:59",
    "398957": "2015-05-27 18:24",
```